build.gradle

```diff
@@ -315,7 +215,8 @@ task assembleOssZipDistribution(dependsOn: assemblyDeps) {
   inputs.files fileTree("${projectDir}/logstash-core/src")
   outputs.files file("${buildDir}/logstash-${project.version}.zip")
   doLast {
-    rubyGradleUtils.rake('artifact:zip_oss')
+      rake(projectDir, buildDir, 'artifact:zip_oss')
+
```

Comment:
```suggestion
```

Reply:
@yaauie, was this an intentional comment?

Reply:
Yes, suggesting removing the additional superfluous whitespace that this PR adds.